Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1908,13 +1908,15 @@ public class BrowserViewController: UIViewController, BrowserViewControllerDeleg
   }
   
   func displayPageZoom(visible: Bool) {
-    if !visible || pageZoomBar != nil {
-      alertStackView.arrangedSubviews.forEach({
-        $0.removeFromSuperview()
-      })
-      
+    if !visible || pageZoomBar != nil {     
+      pageZoomBar?.view.removeFromSuperview()
       updateViewConstraints()
       pageZoomBar = nil
+      
+      if let zoomBarView = pageZoomBar?.view {
```

Comment:
This line never gets hit because `pageZoomBar = nil` is on the line before it.

Reply:
Yeah thats right,  I put it wrong place while editing it. Fix at [e3e29fd](https://github.com/brave/brave-ios/pull/5439/commits/e3e29fdaa59b89cbbb3264998e9f5f4c436854c0)